lib/src/feature_discovery.dart

```diff
@@ -887,21 +888,23 @@ class _Content extends StatelessWidget {
                   children: <Widget>[
                     title == null
                         ? const SizedBox(height: 0)
-                        : Text(title,
-                            style: Theme.of(context)
-                                .textTheme
-                                .title
-                                .copyWith(color: textColor)),
+                        : DefaultTextStyle(
+                          style: Theme.of(context)
+                            .textTheme
+                            .title
+                            .copyWith(color: textColor),
+                          child: title,
+                        ),
                     const SizedBox(height: 8.0),
                     description == null
                         ? const SizedBox(height: 0)
-                        : Text(
-                            description,
-                            style: Theme.of(context)
-                                .textTheme
-                                .body1
-                                .copyWith(color: textColor.withOpacity(0.9)),
-                          ),
+                        : DefaultTextStyle(
+                          style: Theme.of(context)
+                            .textTheme
+                            .body1
+                            .copyWith(color: textColor.withOpacity(0.9)),
```

Comment:
Why is the opacity .9 here and 1 above?

Reply:
It was like that with the implementation of @ayalma and I think it reflects the specs (please take a look at it)

Reply:
Well, in the spec it is just gray, but they do not mention it, so I would just go with what looks best, so leave it for now.

Reply:
It's not grey! It's transparent white, even more transparent than that!